rep-2000.rst

```diff
@@ -715,6 +715,13 @@ Build System Support:
 - cmake
 - setuptools
 
+Rolling Ridley (June 2020 - Ongoing)
+------------------------------------
+
+Rolling Ridley is a rolling development distribution of ROS 2 as described in REP-2000 [8]_.
+
+The target platform for Rolling Ridley will update as new upstream distributions are selected for ROS 2 development.
+As of June 2020, Rolling Ridley targets the same platforms as ROS 2 Foxy Fitzroy.
```

Comment:
I would rather say it targets the platforms of Galactic.

Reply:
To do that we need Galactic in this REP.

Reply:
@cottsay FYI

Reply:
I would suggest that we proceed with reviewing the PR, as-is, to document the fact that as of now, Rolling and Foxy share target platforms, and that when we have a Galactic platform available to target, we update this section of the REP.